geostore/check_stac_metadata/utils.py

```diff
@@ -69,6 +67,18 @@ def maybe_convert_relative_url_to_absolute(url_or_path: str, parent_url: str) ->
     return f"{dirname(parent_url)}/{url_or_path}"
 
 
+def is_s3_url(metadata_url: str) -> bool:
+    return metadata_url[:5] == S3_URL_PREFIX
+
+
+def is_empty(asset_list: List[Dict[str, str]]) -> int:
+    return len(asset_list)
```

Comment:
I don't understand this. Why is this called `is_empty`? And why is `len(asset_list)` pulled out as a function?

Reply:
You are right, but to pull functions out is more easier when mocking it.